.ci/jobs/jobDSL_generator.yml

```diff
@@ -0,0 +1,24 @@
+- job:
```

Comment:
NIT: rename filename to matches the job name `jobdsl-generator` , that's the nomenclature we got in place already :)

Reply:
done